books/free-programming-books-langs.md

```diff
@@ -526,7 +526,6 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Essential C++](https://www.programming-books.io/essential/cpp/) - Krzysztof Kowalczyk, StackOverflow Contributors
 * [Financial Numerical Recipes in C++](https://ba-odegaard.no/gcc_prog/recipes/) - Bernt Arne Ødegaard (PDF)
 * [Fundamentals of C++ Programming](https://web.archive.org/web/20191005170118/https://python.cs.southern.edu/cppbook/progcpp.pdf) - Richard L. Halterman (PDF) *(:card_file_box: archived)*
```

Comment:
```suggestion
* [Fundamentals of C++ Programming](https://web.archive.org/web/20191005170118/https://python.cs.southern.edu/cppbook/progcpp.pdf) - Richard L. Halterman (PDF) *(:card_file_box: archived)*
* [Game Programming Patterns](http://gameprogrammingpatterns.com/contents.html) (HTML)
```
@richelbilderbeek feel free to update this link as I think the suggested link is better than what was there before. 

>**Note** Added this suggestion here since GitHub would not let me make a suggestion for the deleted line below.